.eslintrc.js

```diff
@@ -0,0 +1,112 @@
+module.exports = {
```

Comment:
is it possible to have this file and stylelint file in either "ts" or "json" format ??
(just as a convention as this is a typescript project, so we should not use "js" if possible)

cc @bipuladh

Reply:
are there official type definitions for eslint configuration? Also we will be using third party style guides(abnb) which may not have type definitions for their rules. I have no preference here.

Reply:
json format it is then most other config files are also json